#request
bool req
---
#response
bool success